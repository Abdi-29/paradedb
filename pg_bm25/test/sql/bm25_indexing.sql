CREATE INDEX idxproducts ON products USING bm25 ((products.*));
