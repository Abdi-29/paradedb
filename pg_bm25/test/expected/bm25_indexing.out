CREATE INDEX idxproducts ON products USING bm25 ((products.*));
NOTICE:  table "idxproducts_index" does not exist, skipping
